prisma-fmt/tests/text_document_completion/scenarios/source_schemas_postgres/schema.prisma

```diff
@@ -0,0 +1,12 @@
+generator client {
+    provider = "prisma-client-js"
+    previewFeatures = ["multiSchema"]
+}
+
+datasource db {
+    provider = "postgresql"
+    url      = env("DATABASE_URL")
+    <|>
```

Comment:
Since we have a bug with no completion showing up at the beginning of a line, it would make sense to duplicate this test and also test the beginning of the line?

Reply:
I don't remember if there was an issue / fix for that?

Reply:
https://github.com/prisma/prisma/issues/17000